.zuul.yaml

```diff
@@ -1,4 +1,52 @@
+- job:
+    name: tox-pre-commit
+    parent: tox
+    vars:
+      tox_envlist: pre-commit
+      # NOTE(mnaser): Drop this once zuul/zuul-jobs no longer prefers
+      #               Python 2 by default.
+      tox_prefer_python2: false
```

Comment:
Actually, almost all jobs should run under Python 3. Especially, linters. There should be a matrix of Python versions for pytest tests, and only these should have one Python 2.7 job (until we drop it b/c of EOL).